server/lib/ga-collector.js

```diff
@@ -103,6 +106,30 @@ GACollector.prototype = {
         });
       }
     });
+  },
+  /**
+   * Calculate the queue time of the event.
+   * See https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters?hl=en#qt
+   *
+   * @param {Number} startTime
+   *  Time when metrics object was create on the front-end
```

Comment:
Ever so subtly not true if you make the suggested update from https://github.com/mozilla/fxa-content-server/pull/2926/files#r36738896. `startTime` is really `the time from which all offsets are calculated.`